bisqapps/shared/presentation/src/commonMain/kotlin/network/bisq/mobile/presentation/ui/navigation/README.md

```diff
@@ -0,0 +1,28 @@
+# Navigation Overview
```

Comment:
Excellent @nostrbuddha ! I'll add a link to reference this docs from the main README :)